content/blog/2019-announcing-keynote-speaker-for-pycon-india-2019-asldevi.md

```diff
@@ -0,0 +1,29 @@
+Title: Announcing Keynote Speaker for PyCon India 2019 - Devi A S L
+Date: 2019-08-09 09:00
+Category: 2019
+Slug: announcing-keynote-speaker-for-pycon-india-2019-asldevi
+
+We are happy to announce that **Devi A S L** is a keynote speaker
+for PyCon India 2019!
+
+<!-- PELICAN_END_SUMMARY -->
+
+Devi is an independent software consultant, based in Bangalore.
+She is a long term Pythonista and her work spans diverse domains ranging
+from developing high-performance math libraries, cryptography to
+architecting complex web applications.
+
+Through her work that surpassed over a decade, she has experienced diverse
+work cultures at small startups, MNCs and remote teams where she has been
+through the lows and highs women in tech from different perspectives.
```

Comment:
> she has been through the lows and highs women in tech from different perspectives.

Doesn't this sound incomplete?

Reply:
Yes. Will ask the speaker, about it. Can we merge with a known issue?

Reply:
Sure